src/common/hooks/useTwitchStatus/useTwitchStatus.ts

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+import { TWITCH_STATUS_URL } from '@common';
+
+export const useTwitchStatus = (checkInterval = 60000) => {
```

Comment:
🟢 I wonder whether the project has enough get requests to justify adding a [`useAsync` abstraction](https://github.com/kentcdodds/bookshelf/blob/main/src/utils/hooks.js#L21) to track pending/error statuses.

Reply:
I can check tomorrow and see if the refactor with useAsync is viable.